WL-3992 Added the functionality to disable yellow colour in folderlis…

ouit0408 opened the pull request:
…ting plugin.

Added attribute 'enableHighlight' to the plugins.

nicholaswilson100 commented on library/src/webapp/editor/ckextraplugins/common-wl/js/folder-listing.js:
```diff
@@ -155,6 +156,9 @@ $.fn.folderListing = function(options) {
+  if (!($('.no-highlight').length > 0)){
+    $('[data-folder-listing]').folderListing();
+  }
 });
```
Let's see if we can extract the calling of this plugin so that it's easy for other people to use

nicholaswilson100 replied:
i.e., so that I could, in some other js file, call the plugin with some config, e.g., $('[data-folder-listing]').folderListing('enableHighling':true); or whatever, and it would do all the work for me.

ouit0408 replied:
Implemented the review comment ,updated the commit and been pushed.

ouit0408 committed:
WL-3992 Added the functionality to disable yellow colour in folderlisting plugin.

Added attribute 'enableHighlight' to the plugins which helps in
(de)selecting the yellow highlight.
Removed the automatic bind to data-folder-listing divs which affects the
CK editor folder widget functionality.

buckett commented on library/src/webapp/editor/ckextraplugins/common-wl/js/folder-listing.js:
```diff
@@ -153,8 +154,4 @@ $.fn.folderListing = function(options) {
   });
 };
 
-// automatically bind to data-folder-listing divs
```
I think this will stop the widgets working, although the widget isn't used anywhere else other than this now.

ouit0408 replied:
Yes that was my concern too as removing these lines will stop the functionality of html display of widget on Resources folder.
